examples/lib/defaultHtmlStepUi.js

```diff
@@ -213,7 +213,10 @@ function DefaultHtmlStepUi(_sequencer, options) {
     $(step.imgElement).on('mousemove', _.debounce(() => imageHover(step), 150));
     $(step.imgElement).on('click', (e) => {e.preventDefault(); });
     $stepAll('#color-picker').colorpicker();
-
+    $(step.imgElement).mouseleave(function( event ) {
```

Comment:
Displaying a position of "NULL" when the mouse is not on the current photo may be a bit confusing and/or cluttered — is there any way we could not display the position attributes at all when the mouse is not on the photo? 

We may have to change the location of the element displaying the location so that the text doesn't move around when the location attributes appear and disappear. Maybe move the position text to the top right?

Reply:
True